public/samples/DataFeeds/SVR/broadcaster.ts

```diff
@@ -14,6 +14,8 @@ async function sendBackrunBundle(pendingTx: IPendingTransaction) {
 }
 
 async function buildDummyBackrunTx(pendingTx: IPendingTransaction) {
+  // Your liquidation transaction should send ETH to block.coinbase as the bid
```

Comment:
same here, i don't think we should put this info into the typescript since it needs to be done in solidity.